apps/emqx_cluster_link/src/emqx_cluster_link_sup.erl

```diff
@@ -25,9 +25,9 @@ start_link(LinksConf) ->
 
 init(LinksConf) ->
     SupFlags = #{
-        strategy => one_for_one,
-        intensity => 10,
-        period => 5
+        strategy => one_for_all,
+        intensity => 5,
+        period => 20
```

Comment:
Q: this seems to actually make supervisor restarts more frequent, by making the intensity per time lower.  Is it intentional?
Also, why do we want to restart all children instead of only the crashed one?